viperx/viper.go

```diff
@@ -74,7 +74,7 @@ func WatchConfig(l *logrusx.Logger, o *WatchOptions) {
 		watcherLock.Lock()
 		defer watcherLock.Unlock()
 
-		l.WithField("file", in.Name).
+		defer l.WithField("file", in.Name).
 			WithField("operator", in.Op.String()).
 			WithField("immutables", o.Immutables).
 			Info("The configuration has changed and was reloaded.")
```

Comment:
If you use defer, then `was reloaded` is misleading as it will appear on error as well. So let's change it to:

```suggestion
			Info("A change to the configuration file was processed.")
```